CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Autogenerated by auto2cmake on 2018-03-07 11:49:59
+cmake_minimum_required(VERSION 3.7)
+
+project( matio
+         VERSION 1.5.13
+         LANGUAGES C CXX #Fortran
+)
+
+set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
+
+# Installation options
+include(GNUInstallDirs)
+
+include(cmake/options.cmake)
+include(cmake/thirdParties.cmake)
+include(cmake/compilerOptions.cmake)
+
+set(MATIO_PLATFORM ${CMAKE_SYSTEM_PROCESSOR}-unknown-${CMAKE_SYSTEM_NAME})
+
+include(cmake/run_utils_checks.cmake)
+
+# Setting the include directory for the application to find config.h
+include_directories( ${CMAKE_BINARY_DIR} )
```

Comment:
Best practice these days is to use `target_include_directories` instead, on specific targets.

Reply:
@thejohnfreeman Not being a CMake Pro, can you please help and prepare a PR. Thanks.